cypress/e2e/page/download-pagetests.cy.js

```diff
@@ -21,29 +21,54 @@ describe("tests on download page ui via programmatic state loading and store int
 
             beforeEach(() => {
 
-            });
+                // 1. Open index page
+                cy.visit("/");
 
-            it("", () => {
+                // 2. Load test data
+                cy.loadTestDataIntoStore(p_dataset);
 
-            });
+                // 3. Move to the download page
+                cy.window().its("$nuxt.$router").then(router => {
 
-            it("", () => {
+                    // A. Route to annotation page
+                    router.push({ path: "/download" });
 
-            });
+                    // B. Once routing is complete, set the current page in the nuxt store
+                    // (normally would happen via navigation clicks)
+                    cy.commitToVuexStore("setCurrentPage", "download");
+                });
 
-            it("", () => {
+                // 4. Categories required for this test and the number of required columns for each category
+                const testCriteria = {
 
-            });
+                    categories: [
 
-            it("", () => {
+                        ["Subject ID", 1],
+                        ["Age", 1],
+                        ["Sex", 1]
+                    ]
+                };
 
+                // 5. Load the app with test criteria using the dataset
+                cy.loadAppState("download", p_dataset, testCriteria);
             });
 
-            it("", () => {
+            it("Clicking download button downloads annotated data dictionary JSON", () => {
+
+                // 0. Checking downloads folder state before click
+                cy.task("downloads", "cypress/downloads").then(folderStateBefore => {
```

Comment:
This too should get a section in the internal wiki. 
I also think the comment here would be more helpful if it explained why `cy.task("downloads")` works to check any download folder. I'm assuming `"downloads"` is a builtin / default funtion, and the path is cypress specific? That'd be helpful to understand the test. 

If it's in the wiki, I think a link to the wiki would also do.

Reply:
Added this comment

> NOTE: This runs the NodeJS command 'fs.readdirSync' to poll the 'annotation_tool/cypress/downloads folder' as to its current contents

Reply:
I've added a comment to explain it now and there also will be further documentation of this in https://github.com/neurobagel/documentation/issues/29